### ISSUE TITLE
Header labels

### ISSUE THREAD

**Issue by livstahl** [jakezatecky/react-dual-listbox#22]
Hi Jake,

I am having problems getting the labels above my dual listbox shown. Might be because of the css: ??

.rdl-control-label {
  position: absolute;
  clip: rect(0 0 0 0);
}

Can you help me?

Thanks :-)

**Comment by jakezatecky** [comment]
@livstahl Can you post a screenshot and some sample code?

**Comment by vinayaknadar** [comment]
![image](https://user-images.githubusercontent.com/39020231/50132558-bd633680-02db-11e9-89f1-2709609c0ead.png)

**Comment by jakezatecky** [comment]
In that case you can ignore the `.rdl-control-container` CSS and just do something like

```
.rdl-filter-container .rdl-control-label {
  position: static;
}

.rdl-has-filter .rdl-actions {
    padding-top: 70px;
}
```

The padding might need adjusting.

This might make sense to just add as a feature later.

**Comment by tylergiffel** [comment]
Hi @jakezatecky ,

First, great library. This saved me a good bit of time. I'm having a hell of a time just trying to show some labels above the left listbox and right listbox. I had assumed availableLabel and selectedLabel would display labels above each box but I'm not seeing that.  Am I missing something or would that require an enhancement?

Thanks

**Comment by jakezatecky** [comment]
As this issue would indicate, those labels are hidden but there is a great desire to show these labels. Therefore, I will be making an addition that allows these labels to be easily shown. I likely will be adding `showHeaderLabels` and `showFilterLabels` properties.

**Comment by oscarlnetoo** [comment]
Hi @jakezatecky,

Is it possible to remove the word "Filter" in the label?

![image](https://user-images.githubusercontent.com/18273311/52008352-993c6d00-24b7-11e9-8028-190312fd7325.png)

Can you help me?

Btw, thank you for the lib. It is helping a lot.

**Comment by jakezatecky** [comment]
Hello @oscarlnetoo. Unfortunately, this was hard-coded into the component. There is a `lang` property, but that does not currently influence this header.

This will be remedied with the v2.0.0 release, which is the next slated release. It is my intention add keys to `lang` such that you can specify whatever strings for those headers.

**Comment by jakezatecky** [comment]
Feature implemented in dev branch. Property `showHeaderLabels` will be available in v2.0.0 and `availableHeader` and `selectedHeader` will be specifiable in the `lang` property. The header labels have been moved above the filter boxes while the filter label will be suppressed. This should satisfy the issues brought up in this thread.
